bugsnag-android-core/src/test/java/com/bugsnag/android/internal/journal/BugsnagJournalEventMapperTest.kt

```diff
@@ -40,6 +42,10 @@ class BugsnagJournalEventMapperTest {
                 "foo" to fooSection
             )
         )
+
+        journalMap = minimalJournalMap.plus(
+            mapOf("context" to "ExampleActivity")
+        )
```

Comment:
Would this not work here?

```suggestion
        journalMap = minimalJournalMap + ("context" to "ExampleActivity")
```

Leaning on:

```kotlin
operator fun <K, V> Map<out K, V>.plus(
    pair: Pair<K, V>
): Map<K, V>
```